src/app/edit-create-study-set/edit-create-study-set.component.html

```diff
@@ -0,0 +1,59 @@
+<head>
+  <title>Edit Create Study Set</title>
+</head>
+
+
+<body>
+<div class = "body-div">
+  <div class = "upper-components">
+    <div class ="first-row">
+      <p class = "page-title">Create Study Set</p>
```

Comment:
change `p` to `h1` and set a global style in `src/app/styles.scss` for all headings `h1`.